instrumentation/opentelemetry-instrumentation-redis/src/opentelemetry/instrumentation/redis/__init__.py

```diff
@@ -79,10 +79,9 @@ def _traced_execute_command(func, instance, args, kwargs):
         name, kind=trace.SpanKind.CLIENT
     ) as span:
         if span.is_recording():
-            span.set_attribute("service", tracer.instrumentation_info.name)
             span.set_attribute(_RAWCMD, query)
             _set_connection_attributes(span, instance)
-            span.set_attribute("redis.args_length", len(args))
+            span.set_attribute("db.redis.args_length", len(args))
```

Comment:
Similar to `db.redis.database_index`; DB specific attribute keys should be in the form of `db.{identifier}.{}`.